.deps/dev.md

```diff
@@ -680,13 +741,25 @@
 | [`please-upgrade-node@3.2.0`](git+https://github.com/typicode/please-upgrade-node.git) | MIT | clearlydefined |
 | [`portfinder@1.0.26`](git@github.com:http-party/node-portfinder.git) | MIT | clearlydefined |
 | [`posix-character-classes@0.1.1`](https://github.com/jonschlinkert/posix-character-classes.git) | MIT | [CQ17757](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17757) |
+| [`postcss-html@0.36.0`](git+https://github.com/gucong3000/postcss-html.git) | MIT | clearlydefined |
+| [`postcss-jsx@0.36.4`](git+https://github.com/gucong3000/postcss-jsx.git) | MIT | clearlydefined |
+| [`postcss-less@3.1.4`](https://github.com/shellscape/postcss-less.git) | MIT | clearlydefined |
+| [`postcss-markdown@0.36.0`](git+https://github.com/gucong3000/postcss-markdown.git) | MIT | clearlydefined |
+| [`postcss-media-query-parser@0.2.3`](git+https://github.com/dryoma/postcss-media-query-parser.git) | MIT | clearlydefined |
 | [`postcss-modules-extract-imports@2.0.0`](https://github.com/css-modules/postcss-modules-extract-imports.git) | ISC | clearlydefined |
 | [`postcss-modules-local-by-default@3.0.2`](https://github.com/css-modules/postcss-modules-local-by-default.git) | MIT | clearlydefined |
 | [`postcss-modules-scope@2.2.0`](https://github.com/css-modules/postcss-modules-scope.git) | ISC | clearlydefined |
 | [`postcss-modules-values@3.0.0`](git+https://github.com/css-modules/postcss-modules-values.git) | ISC | clearlydefined |
+| [`postcss-reporter@6.0.1`](https://github.com/postcss/postcss-reporter.git) | MIT | clearlydefined |
+| [`postcss-resolve-nested-selector@0.1.1`](Unknown) | MIT | clearlydefined |
+| [`postcss-safe-parser@4.0.2`](https://github.com/postcss/postcss-safe-parser.git) | MIT | clearlydefined |
+| [`postcss-sass@0.4.4`](https://github.com/AleshaOleg/postcss-sass.git) | MIT | clearlydefined |
+| [`postcss-scss@2.1.1`](https://github.com/postcss/postcss-scss.git) | MIT | clearlydefined |
 | [`postcss-selector-parser@6.0.2`](https://github.com/postcss/postcss-selector-parser.git) | MIT | clearlydefined |
+| [`postcss-sorting@5.0.1`](https://github.com/hudochenkov/postcss-sorting.git) | MIT | clearlydefined |
+| [`postcss-syntax@0.36.2`](git+https://github.com/gucong3000/postcss-syntax.git) | MIT |  |
```

Comment:
Could you provide a link for the needed CQ?

Reply:
it's a generated file. All changes added manually will be lost after running license tools.